packages/frontend/index.d.ts

```diff
@@ -4,7 +4,7 @@ interface ArticleProps {
     config: ConfigType;
 }
 
-type Pillar = 'news' | 'opinion' | 'sport' | 'culture' | 'lifestyle';
+type Pillar = 'news' | 'opinion' | 'sport' | 'culture' | 'lifestyle' | 'labs';
```

Comment:
Can we add a comment here about labs being a 'special/fake' pillar or something just so future us understand things.

Reply:
Will do!